src/api/config/options.yml.example

```diff
@@ -183,6 +183,14 @@ default: &default
   #    persistent: true
   #    passive: true
 
+  # For storing rails metrics in influxdb
```

Comment:
This file is for Admins. I would suggest:

`# For sending application performance metrics to a influx time series database`